pkg/flowaggregator/flowaggregator.go

```diff
@@ -15,25 +15,250 @@
 package flowaggregator
 
 import (
+	"fmt"
+	"hash/fnv"
 	"net"
 	"time"
 
-	"github.com/vmware/go-ipfix/pkg/registry"
+	ipfixentities "github.com/vmware/go-ipfix/pkg/entities"
+	ipfixintermediate "github.com/vmware/go-ipfix/pkg/intermediate"
+	ipfixregistry "github.com/vmware/go-ipfix/pkg/registry"
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/ipfix"
+	"github.com/vmware-tanzu/antrea/pkg/util/env"
+)
+
+var (
+	IANAInfoElements = []string{
```

Comment:
does `IANAInfoElements` need to be public?

Reply:
Changed IANAInfoElements, IANAReversedInfoElements and AntreaInfoElements into private.